03_launch_mgmt_cluster.sh

```diff
@@ -169,7 +169,7 @@ EOF
   fi
 
   # Copy the generated configmap for ironic deployment
-  if [[ ${BMOBRANCH} == "v0.1.2"  ]]; then # BMORELEASE until v0.1.2 used the old path TODO(mboukhalfa) can be removed after new bmo release
+  if [[ ${BMOBRANCH} == "v0.1.2" || ${BMOBRANCH} == "v0.1.1" ]]; then # BMORELEASE until v0.1.2 used the old path TODO(mboukhalfa) can be removed after new bmo release
```

Comment:
Why not just match anything starting with `v0.1.` ? 

Also, if you do `||`, it needs to go outside `[`, ie. `[[ .. ]]  || [ ... ]]`